build_tools/github_actions/runner/gcp/update_instance_groups.py

```diff
@@ -93,6 +93,139 @@ def get_migs(self, *, region, type, group, prefix, modifier=None):
         return migs
 
 
+def update_mig(*, templates_client, migs_client, mig, args):
```

Comment:
I assume this part is simply moved out from the main?

Reply:
Yes. Moved without change